src/Import/Remote/Composer.php

```diff
@@ -75,10 +75,10 @@ public function remove(): void
     /**
      * @param string[] $args
      */
-    public function run(string $composerJsonFilePath, string $vendorDirectory, array $args, callable|OutputInterface $callback, bool $allowInteraction = false): void
+    public function run(string $composerJsonFilePath, string $workingDirectory, array $args, callable|OutputInterface $callback, bool $allowInteraction = false): void
     {
         $args[] = '--working-dir';
-        $args[] = \dirname($vendorDirectory);
+        $args[] = \dirname($workingDirectory);
```

Comment:
You should drop dirname here since we don't pass the vendor directory (and it should be fixed in the install function to pass the correct working dir)